install.sh

```diff
@@ -67,10 +89,10 @@ echo -e "|---${GREEN}Pacstall Installer${NC}---|"
 echo -e "|------------------------|"
 
 if [[ ${GITHUB_ACTIONS} != "true" ]]; then
-	if ! ping -c 1 github.com &> /dev/null; then
-		fancy_message error "You seem to be offline"
+	check_url "https://github.com" || check_url "https://gitlab.com" || {
```

Comment:
This should only be to GitHub, as that's where we pull scripts from, so if githubs down but gitlab works, it doesn't matter

Reply:
We support custom repos from gitlab

Reply:
We do, but our scripts are located on GitHub at the moment, meaning that this ping to gitlab doesn't matter, because if githubs down and gitlab isn't, we still can't pull scripts

Reply:
Well, what if the user has removed the official repo and is only using a gitlab custom repo